src/main/server.js

```diff
@@ -1,81 +1,5 @@
-import express, { json, urlencoded } from 'express';
-import admin from 'firebase-admin';
-import serviceAccount from '../../../claves/metal-slug-maker-firebase-adminsdk-8dsbv-e517ebfe1a.json';
```

Comment:
Sorry for re-request this changes but it could throw an error. Please change the folder here as well

Reply:
this line doesn't exist anymore. notice that is highlighted in red.
this file only contains the green lines. 
it imports app.js and set the listener port

Reply:
My bad, sorry